README.md

```diff
@@ -157,19 +157,29 @@ class LitAutoEncoder(L.LightningModule):
         optimizer = torch.optim.Adam(self.parameters(), lr=1e-3)
         return optimizer
 
+    # -------------------
+    # Step 2: Define data
+    # -------------------
+    def prepare_data(self):
+        tv.datasets.MNIST(".", download=True)
+
+    def setup(self, stage):
```

Comment:
There's a small bug here in that the dataset will get created twice, as this will get called once for training and once for validation.

We could add ```if hasattr(self, "train_dataset"): return``` but I didn't in case we consider that ugly for a README example